team_picker/routes/teams.js

```diff
@@ -0,0 +1,94 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../db/client");
+
+router.get('/', (req, res) => {
+  res.render('main'); 
+});
+
+router.get("/new", (req, res) => {
+  res.render("new");
+});
+
+// Enter new cohort
+router.post("/new", (req, res) => {
+  console.log("Saving data to database from new teams")
+  knex("cohorts")
+    .insert({
+      name: req.body.name,
+      logoUrl: req.body.logoUrl,
+      members: req.body.members
+    })
+    .returning("*")
+    .then(cohorts => {
+      res.render('show', { cohort: cohorts[0], pickerMethod: undefined, quantity: 0 });
+    });
+});
+
+// List of all the cohorts
+router.get('/cohorts', (req, res) => {
+  knex("cohorts")
+    .orderBy("name")
+    .then(cohorts => {
+      res.render("index", { cohorts });
+    });
+});
+
+// Display a specific cohort and team picker
+router.get('/teams/:id', (req, res) => {
+  const id = req.params.id;
+  knex("cohorts")
+  .where("id", id)
+  .first()
+  .then(cohort => {
+    res.render("show", { cohort, pickerMethod: undefined, quantity: 0 });
+  });
+})
+router.post("/teams", (req, res) => {
```

Comment:
You probably want this to be `/teams/:id`, instead of passing the `id` in the `req.body`. You're dealing with the `:id` team "resource", so the URL should reflect that.

Reply:
@rosadiaz, Just to clarify, what @diazruy is talking about is that there is a pattern for handling requests. https://www.youtube.com/watch?v=7YcW25PHnAA